web/src/util/index.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/prefer-default-export */
+
+export const STATES = [
+  { id: 'al', name: 'Alabama' },
+  { id: 'ak', name: 'Alaska' },
+  { id: 'az', name: 'Arizona' },
+  { id: 'ar', name: 'Arkansas' },
+  { id: 'ca', name: 'California' },
+  { id: 'co', name: 'Colorado' },
+  { id: 'ct', name: 'Connecticut' },
+  { id: 'de', name: 'Delaware' },
+  { id: 'fl', name: 'Florida' },
+  { id: 'ga', name: 'Georgia' },
+  { id: 'hi', name: 'Hawaii' },
+  { id: 'id', name: 'Idaho' },
+  { id: 'ia', name: 'Iowa' },
+  { id: 'il', name: 'Illinois' },
+  { id: 'in', name: 'Indiana' },
+  { id: 'ks', name: 'Kansas' },
+  { id: 'ky', name: 'Kentucky' },
+  { id: 'la', name: 'Louisiana' },
+  { id: 'me', name: 'Maine' },
+  { id: 'md', name: 'Maryland' },
+  { id: 'ma', name: 'Massachusetts' },
+  { id: 'mi', name: 'Michigan' },
+  { id: 'mn', name: 'Minnesota' },
+  { id: 'ms', name: 'Mississippi' },
+  { id: 'mo', name: 'Missouri' },
+  { id: 'mt', name: 'Montana' },
+  { id: 'ne', name: 'Nebraska' },
+  { id: 'nc', name: 'North Carolina' },
+  { id: 'nd', name: 'North Dakota' },
+  { id: 'nh', name: 'New Hampshire' },
+  { id: 'nj', name: 'New Jersey' },
+  { id: 'nm', name: 'New Mexico' },
+  { id: 'nv', name: 'Nevada' },
+  { id: 'ny', name: 'New York' },
+  { id: 'oh', name: 'Ohio' },
+  { id: 'ok', name: 'Oklahoma' },
+  { id: 'or', name: 'Oregon' },
+  { id: 'pa', name: 'Pennsylvania' },
+  { id: 'ri', name: 'Rhode Island' },
+  { id: 'sc', name: 'South Carolina' },
+  { id: 'sd', name: 'South Dakota' },
+  { id: 'tn', name: 'Tennessee' },
+  { id: 'tx', name: 'Texas' },
+  { id: 'ut', name: 'Utah' },
+  { id: 'vt', name: 'Vermont' },
+  { id: 'va', name: 'Virginia' },
+  { id: 'wa', name: 'Washington' },
+  { id: 'dc', name: 'Washington, DC' },
+  { id: 'wv', name: 'West Virginia' },
+  { id: 'wi', name: 'Wisconsin' },
+  { id: 'wy', name: 'Wyoming' }
+];
```

Comment:
While we should let the header go for now, this list should include the District of Columbia, American Samoa, Guam, Northern Mariana Islands, Puerto Rico, and US Virgin Islands.  I think that's all, but we should ask @jeromeleecms or @NAretakis to make sure.